receiver/loadgenreceiver/logs.go

```diff
@@ -130,3 +145,7 @@ func (ar *logsGenerator) nextLogs() plog.Logs {
 
 	return nextLogs
 }
+
+func (ar *logsGenerator) isDone() bool {
+	return ar.samples.LoopCount() > 0 && ar.samples.LoopCount() >= ar.cfg.Logs.MaxReplay
```

Comment:
Why do we need the >0 check as well as comparison?

Reply:
:facepalm: meant to type ar.cfg.Logs.MaxReplay for unlimited replays.